prefix-forks-with-owner.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+function printError() {
+	echo "$@"
+	exit 1
+}
+
+if [ "$#" -ne 1 ]; then
+	name="$(basename "$0")"
+	printError "Usage: $name <user>"
+fi
+user=$1
+
+which jq >/dev/null || printError "please install jq"
+which gh >/dev/null || printError "please install GitHub CLI"
+
+gh auth status >/dev/null || printError "Please configure GitHub CLI with: gh auth login"
+
+gh repo list --fork --limit 1000 --json name,parent "$user" | jq -r '.[] | [.name, .parent.owner.login, .parent.name] | @tsv' | while read -r repo owner orepo; do
```

Comment:
nbd but the 2 scripts are essentially copy paste of the same start, maybe make one a subcommand or flag?

Reply:
Yes, that's right.

I created one by copy pasting the first one 😅😬

I'm unsure what I could do now.

Maybe I will merge as this, after fixing some feedbacks.

Reply:
I will address this with a complete rewrite

https://github.com/ccoVeille/github-forks-sanitizer/issues/2